### ISSUE TITLE
about system.prop

### ISSUE THREAD

**Issue by KevinMX** [kadabra10/android_device_meizu_m2note#1]
I suggest ro.sf.lcd_density=480 for better display.
Of cource 320 for better performance and battery save.
And need to change ro.sf.hwrotation=0 to 180, or the screen will be upside down.

**Comment by Moyster** [comment]
Yes it's a device tree for CM, check the cm.mk file ( omni.mk is omnirom, etc..) ;)
Kadabra10 made a m571-cm github page, don't know if he preffers PR/Issues on this page or the m571-cm repo tho ?
